lib/evmone_precompiles/bls.cpp

```diff
@@ -68,4 +70,28 @@ void store_result(uint8_t _rx[64], uint8_t _ry[64], const blst_p1* out) noexcept
 
     return true;
 }
+
+[[nodiscard]] bool g1_mul(uint8_t _rx[64], uint8_t _ry[64], const uint8_t _x[64],
+    const uint8_t _y[64], const uint8_t _c[32]) noexcept
+{
+    blst_scalar scalar;
+    blst_scalar_from_bendian(&scalar, _c);
```

Comment:
I wonder what it actually does. Because the later `blst_p1_mult` just uses the bytes. Are they the same bytes as the input?